module/VuFind/src/VuFind/Db/Entity/Search.php

```diff
@@ -38,94 +38,138 @@
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org/wiki/development:plugins:database_gateways Wiki
  *
- * @ORM\Table(name="search", indexes={@ORM\Index(name="folder_id", columns={"folder_id"}), @ORM\Index(name="notification_base_url", columns={"notification_base_url"}), @ORM\Index(name="notification_frequency", columns={"notification_frequency"}), @ORM\Index(name="session_id", columns={"session_id"}), @ORM\Index(name="user_id", columns={"user_id"})})
+ * @ORM\Table(name="search",
+ * indexes={@ORM\Index(name="folder_id", columns={"folder_id"}),
+ * @ORM\Index(name="notification_base_url",  columns={"notification_base_url"}),
+ * @ORM\Index(name="notification_frequency", columns={"notification_frequency"}),
+ * @ORM\Index(name="session_id",             columns={"session_id"}),
+ * @ORM\Index(name="user_id",                columns={"user_id"})})
  * @ORM\Entity
  */
 class Search implements EntityInterface
 {
     /**
+     * Unique ID.
+     *
      * @var int
      *
-     * @ORM\Column(name="id", type="bigint", nullable=false, options={"unsigned"=true})
+     * @ORM\Column(name="id",
+     *          type="bigint",
+     *          nullable=false,
+     *          options={"unsigned"=true}
+     * )
      * @ORM\Id
      * @ORM\GeneratedValue(strategy="IDENTITY")
      */
     protected $id;
 
     /**
+     * User id.
+     *
      * @var int
      *
      * @ORM\Column(name="user_id", type="integer", nullable=false)
      */
     protected $userId = '0';
 
     /**
+     * Session id.
+     *
      * @var string|null
      *
      * @ORM\Column(name="session_id", type="string", length=128, nullable=true)
      */
     protected $sessionId;
 
     /**
+     * Folder id.
+     *
      * @var int|null
      *
      * @ORM\Column(name="folder_id", type="integer", nullable=true)
      */
     protected $folderId;
 
     /**
+     * Created date.
+     *
      * @var \DateTime
      *
-     * @ORM\Column(name="created", type="datetime", nullable=false, options={"default"="2000-01-01 00:00:00"})
+     * @ORM\Column(name="created",
+     *          type="datetime",
+     *          nullable=false,
+     *          options={"default"="2000-01-01 00:00:00"}
+     * )
      */
     protected $created = '2000-01-01 00:00:00';
 
     /**
+     * Title.
+     *
      * @var string|null
      *
      * @ORM\Column(name="title", type="string", length=20, nullable=true)
      */
     protected $title;
 
     /**
+     * Saved.
+     *
```

Comment:
This should be a boolean rather than an int. Looks like this is int(1) instead of tinyint(1) in the MySQL schema. We should probably either open a PR to address this, or add the changes to [#2689](https://github.com/vufind-org/vufind/pull/2689) and rename that to "Fix boolean types in database" (or something similar).